### PR TITLE
XP updates

### PR DESCRIPTION
**Pull request by cmaxwe11**
Updated the XP playbook with some links to SC and the explorer. Also updated the 'how does it work' xp playbook to link to the XP playbook for more detail.

### CODE REVIEW

File: docs/playbooks/mgxp-metagame-experience-points.md [hunk 1]
```diff
@@ -57,3 +57,7 @@ Nope! Reactions are weighted different based both on your role and the emoji.
 ![](https://i.imgur.com/6o4PpVT.png)
 
 ![](https://i.imgur.com/W3MNipm.jpg)
+
+## Where can I see how much XP I've earned?
+
+You have a few different ways to track your XP. The simplest way is to check out your MyMeta profile, your XP is listed there. If you want to dig a little deeper, you can look directly at the [Sourcecred](https://xp.metagame.wtf/#/explorer) instance, or dig in with this handy [explorer](https://observablehq.com/@dysbulic/metagame-contributions-explorer) that @dysbulic built.
```
Comment (luxumbra) [inline docs/playbooks/mgxp-metagame-experience-points.md:63]: I'd probably add 'instance' to the anchor text here as you're linking to our instance and not just Sourcecred. `[Sourcecred instance](https://xp.metagame.wtf/#/explorer)`

**Review by luxumbra — CHANGES_REQUESTED** [review]
The change in anchor text is a picky thing but imho is worth doing.

File: docs/playbooks/mgxp-metagame-experience-points.md (hunk 1 above)
Comment (cmaxwe11) [reply docs/playbooks/mgxp-metagame-experience-points.md]: I agree! Updated.

**Review by luxumbra — APPROVED** [review]
LGTM!

**Pull request merged**